### DISCUSSION TITLE
V3 housing 3d model

### DISCUSSION THREAD

**Discussion by jdugan4859** [gtxaspec/wz_mini_hacks#253] [General]
Does anyone know if there is a 3d model out there of the camera housing? I’ve been looking at ways to integrate the ethernet adapter into the inside of the camera. It’s really close to fitting, but just not there. If I could get a 3d model, and make some changes to it, and have a new housing 3d printed, I think it would work pretty well.

**Comment by capncybo** [comment]
That is a Fantastic Idea!!!

**Reply by jdugan4859** [reply]
I didn’t have any luck finding a model, so I grabbed a set of calipers, and jumped in to AutoCAD to do the initial modeling. I’m working in all 2d right now, gonna start going over to 3d tomorrow, and then see what I can do about getting it into something I can print off of. The changes will ve, fairly minimum, the back of the case is going to have to be removable, the lengthen of the housing is going to get a bit longer, and the overall height is going to get taller. Not so much that it would stand out, but it will I have an issue with the LAN port I am working on, but I think I’m well on my way.

once I get the file done, does anyone here 3d print and would run a test print to check for figment?
